lib/CredentialIssuancePurpose.js

```diff
@@ -8,11 +8,50 @@ const {AssertionProofPurpose} = require('jsonld-signatures').purposes;
  * Creates a proof purpose that will validate whether or not the verification
  * method in a proof was authorized by its declared controller for the
  * proof's purpose.
- *
- * Required params:
- * - `controller`
  */
 class CredentialIssuancePurpose extends AssertionProofPurpose {
+  /**
+   * @param {object} controller - the description of the controller, if it
+   *   is not to be dereferenced via a `documentLoader`. Required.
```

Comment:
This documentation is confusing; can't `controller` be omitted if the `documentLoader` will handle it? And if so, shouldn't the check below that requires `controller` be removed?

Reply:
Right, ok, so, that's part of the question I had.